frontend/src/components/PPViewer/types/index.ts

```diff
@@ -0,0 +1,9 @@
+interface IPPViewer {
+  link: string;
+}
+
+interface IPPViewerContainer {
+  link: string;
+}
```

Comment:
should we duplicate these interfaces since they are the same?

Reply:
let's replace it with one IPPViewerProps interface

Reply:
But what if logics will be added later? Will we have to separate them again?)

Reply:
then we can add a new interface extending this one (if it will be necessary) and add all the required properties without repeating.
like: 

```
interface IPPViewerProps {
  link: string;
}

interface IPPViewerContainerProps extends IPPViewerProps {
  newProperty: newPropertyType;
}
```